static/app/views/performance/newTraceDetails/traceMetadataHeader.tsx

```diff
@@ -20,16 +25,344 @@ interface TraceMetadataHeaderProps {
   traceSlug: string;
 }
 
+export enum TraceViewReferrers {
```

Comment:
been trying to use more const enums where possible since they compile down better
```suggestion
export const enum TraceViewReferrers {
```